README.md

```diff
@@ -20,6 +20,10 @@ Server as a document database and cache while retaining store-specific features
 of Spring Data Couchbase are a POJO centric model for interacting with a Couchbase Server Bucket and easily writing a
 repository style data access layer.
 
+Integration tests require a couchbase server with a bucket name "protected" with "password" as the password set.
```

Comment:
can you work with oliver to get the int system updated to 5.0.1?

Reply:
or.. check with them if docker is available on the CI machines, I don't know if that is supported by them. just so the datacouch CI system keeps working

Reply:
Hmm sure but I dont think their CI runs integration tests yet.